libobs/obs-nix-x11.c

```diff
@@ -0,0 +1,1270 @@
+/******************************************************************************
```

Comment:
git mv this file from its source to preserve history and ease my sanity trying to review changes.

Reply:
By reading your comment, and the commit, I thought I forgot to use `git mv`. But I just redid this commit using `git mv`, and the result looks the same. Sorry, looks like git falls back to the rewrite strategy (given the big diff) and the rename operation is rendered useless :(

Reply:
Well that sucks. Thanks for taking a look.